src/sentry/static/sentry/app/components/events/interfaces/breadcrumbs/data/summary.tsx

```diff
@@ -38,6 +38,7 @@ class Summary extends React.Component<Props, State> {
     }
 
     return Object.keys(kvData)
+      .reverse()
```

Comment:
Do we want to reverse all of them? Or just order of "from" and "to" on navigation breadcrumb?

Reply:
yes it will reverse all of them, but I think everything is fine because we are getting the correct backend order and for some reason this is being reversed on the client side. I didn't want to waste a lot of time on this, but what I've realized so far is that the data has been grouped into a Proxy and when we do Object.keys the order is reversed.